addons/account/views/report_invoice.xml

```diff
@@ -249,10 +249,15 @@
                     </p>
                     <div id="qrcode" t-if="o.display_qr_code and o.amount_residual > 0">
                         <t t-set="qr_code_url" t-value="o._generate_qr_code()"/>
-                        <p t-if="qr_code_url">
```

Comment:
No big deal, but let's not change the invoice layout for everyone outside of India. 

At the moment the QR code is below the text 'Scan me with your banking app', let's keep it this way. 

The POs are looking into how they want the invoices to look, and at some point they will come up with improvements, but for the moment having the QR code under the text is intentional IMO. 

On the other hand, I think your design makes sense for Indian companies wanting to get paid via UPI.

Reply:
@antoine162 It might make sense in generic as well.  (it is master)  (could we pass this to some PO?  @tsb-odoo )